Module-generation/scrape.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import requests
+from github import Github
+
+IndexDir="./meta-data/"
+os.mkdir(IndexDir)
+
+accessToken=os.environ.get('GIT_CREDS')
+g = Github(accessToken)
+
+
+user= g.get_user('Terasology')
+repo = user.get_repo("Index")
+
+contents = repo.get_contents("/distros/omega/gradle.properties")
+x=contents.decoded_content.decode()
+
+# Filter Modules and Convert it to array
+f = open('Modules.txt', 'w')
+f.write(x)
+f.close()
+f = open("Modules.txt", "r")
+lines=f.readlines()
+line=lines[1]
+word1='extraModules='
+word2='\n'
+wordreplace=line.replace(word1, "")
+wordreplace=wordreplace.replace(word2,"")
+Modules=wordreplace.split(",")
+
+
+# Collect Modules
+for i in range(0,len(Modules)):
+   if(i<len(Modules)):
+      repository= user.get_repo(Modules[i])
+      con=repository.get_contents("")
+      for content_file in con:
+          if(content_file.path=="module.txt"):
+              getModuleCon=repository.get_contents("module.txt")
+              getData=getModuleCon.decoded_content.decode()
+              parseData=json.loads(getData)
+              ModuleName=parseData['id']
+              ModuleDir=os.mkdir(IndexDir+ModuleName)
+              ModuleDirSrc=IndexDir+ModuleName
+
+              # write module.txt in module directory
+
+              ModuleFile= open(ModuleDirSrc+"/module.txt", "a")
+              ModuleFile.write(getData)
+              ModuleFile.close()
+
+              # search for readme files
+              readmeFound=0
+              for Readme in con:
+                  if(Readme.path=="README.md" or Readme.path=="README.MD" or Readme.path=="README.markdown"):
```

Comment:
as discussed in the last meeting: stick with one and fall back to a default text if that's not present
no need to overcomplicate things here.

Reply:
Did that during frontmatter generation (frontmatter.py line no 93 to 97)